cedar-policy-validator/src/type_error.rs

```diff
@@ -267,9 +277,10 @@ pub struct TypesMustMatch {
 
 /// Structure containing details about a missing attribute error.
 #[derive(Debug, Hash, Eq, PartialEq)]
-pub struct MissingAttribute {
+pub struct UnsafeAttributeAccess {
     missing: String,
     suggestion: Option<String>,
+    may_exist: bool,
```

Comment:
docs on this?  I presume the interpretation is that, if `may_exist` is true, the `missing` attribute might actually exist we just don't know for sure?